Header header
navig/MsgOdometry odometry
uint8 camera_type
int32 frame_number
vision/MsgGate[] gate